D/difmap/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "difmap"
+# difmap version is 2.5e, mapped here to numerical value of 2.5.5
+version = v"2.5.5"
+
+sources = [ArchiveSource("ftp://ftp.astro.caltech.edu/pub/difmap/difmap2.5e.tar.gz", "457cd77c146e22b5332403c19b29485388a863ec494fff87137176396fc6a9ff")]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+cd uvf_difmap
+sed -i 's|^PGPLOT_LIB="-lpgplot -lX11"|PGPLOT_LIB="-L/usr/X11R6/lib -Xlinker -R/usr/X11R6/lib -lpgplot -lX11"|' configure
```

Comment:
All those `/usr` paths are definitely wrong

Reply:
I guess it was taken from some older scripts of mine, and this edit is indeed not needed.